src/styles/blocks/card.scss

```diff
@@ -0,0 +1,74 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__description {
+    margin-bottom: 16px;
+
+    .card__name {
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+      color: $name-color;
+      margin-top: 0;
+      margin-bottom: 4px;
+    }
+
+    .card__code {
+      margin-block: 0;
+      font-size: 10px;
+      line-height: 14px;
+      color: #616070;
+    }
+  }
+
+  .card__rating {
+    @include spacebetweenblock;
+
+    align-items: center;
+    width: 100%;
+    margin-bottom: 24px;
+
+    .card__reveiws {
+      margin-block: 0;
+      font-size: 10px;
+      line-height: 14px;
+      color: $name-color;
+    }
+  }
+
+  &__price {
+    @include spacebetweenblock;
+
+    width: 100%;
+    margin-bottom: 16px;
+
+    .card__priceName {
```

Comment:
According to BEM naming rules (there are a bit different variants, but we talk about those which were described in theory and videos. Also, you can take a look at rules [here](https://en.bem.info/methodology/naming-convention/)) separate words in name should be separated with `-`.

Also, no need to put nested in HTML file elements inside their parent in SCSS. Here it's all separate BEM elements, so you should write it like that:

```
&__price {
  // styles here
}

&__price-name {
  // styles here
}
```